lumicks/pylake/detail/utilities.py

```diff
@@ -30,6 +30,39 @@ def first(iterable, condition=lambda x: True):
     return next(x for x in iterable if condition(x))
 
 
+def replace_key_aliases(dictionary, *key_alias_lists):
+    """Replace key aliases by a specific one.
+
```

Comment:
I think it would be useful to add a bit more explanation here as the function is generic (which I like) but a bit abstract-sounding, at least from the argument names. Maybe something like:
```suggestion
    """Replace key aliases by a specific one.
    
       This function is especially useful for validating user-supplied plotting kwargs
       against pylake-supplied default plotting parameters. For instance, a user could 
       supply either `color="red"` or `c="red"` to specify the line color. 

```

Reply:
Yeah, that's a good point. I bet our future selves would be thankful for that :D